image/manifest.go

```diff
@@ -117,7 +117,7 @@ func (m *manifest) unpack(w walker, dest string) error {
 			return err
 		}
 	}
-	return nil
+	return fmt.Errorf("no validate layers found")
```

Comment:
I think the error should be like this one: https://github.com/coolljt0725/image-spec/blob/290cb54341fdce2b746a05fc164203843f41dd7e/image/manifest.go#L113